src/main/kotlin/no/skatteetaten/aurora/boober/model/AuroraConfig.kt

```diff
@@ -19,13 +20,14 @@ data class AuroraConfig(val auroraConfigFiles: List<AuroraConfigFile>, val affil
 
         @JvmStatic
         fun fromFolder(folder: File): AuroraConfig {
+            val mapper = jacksonYamlObjectMapper()
             val files = folder.walkBottomUp()
                     .onEnter { !setOf(".secret", ".git").contains(it.name) }
-                    .filter { it.isFile }
+                    .filter { it.isFile && listOf("json", "yaml").contains(it.extension) }
```

Comment:
Note that this change will also allow the repo to contain dofiles and markdown/asciidoc files since we here explicitly only allow yaml and json files to be AuroraConfig files.

Reply:
sounds useful